alma/lib/Model/PaymentData.php

```diff
@@ -170,6 +170,11 @@ public static function dataFromCart($cart, $context, $feePlans, $forPayment = fa
         $customerData['state_province'] = State::getNameById((int) $idStateBilling);
         $customerData['country'] = Country::getIsoById((int) $billingAddress->id_country);
 
+        if ($billingAddress->company) {
+            $customerData['is_business'] = true;
+            $customerData['business_name'] = $billingAddress->company;
```

Comment:
are we missing the siret number ?

Reply:
Thanks @PaulRouss3l 
We haven't siret number by default in Prestashop, but VAT number. I don't think Romain want VAT number in `business_id_number` field.

Reply:
I'm afraid that the B2B side of the API won't be able to work properly without a `business_id_number` though :/

Reply:
@PaulRouss3l The customer will be obliged to complete the field in the checkout page. 
I can see with Romain if I have to put VAT number in `business_id_number`